finalitygadget/finalitygadget.go

```diff
@@ -252,46 +252,22 @@ func (fg *FinalityGadget) QueryBlockRangeBabylonFinalized(
 	return finalizedBlockHeight, nil
 }
 
-/* QueryBtcStakingActivatedTimestamp returns the timestamp when the BTC staking is activated
- *
- * - We will check for k deep and covenant quorum to mark a delegation as active
```

Comment:
why the entire blockstring removed? should we move it to the new function you created?

Reply:
We have this in the interface